staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -53,11 +53,12 @@ type KubeSchedulerConfiguration struct {
 	// ClientConnection specifies the kubeconfig file and client connection
 	// settings for the proxy server to use when communicating with the apiserver.
 	ClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"clientConnection"`
-	// HealthzBindAddress is the IP address and port for the health check server to serve on,
-	// defaulting to 0.0.0.0:10251
+
+	// Note: Both HealthzBindAddress and MetricsBindAddress fields are deprecated
```

Comment:
Put the deprecated note in each field's go comment. And say that only empty or port 0 is allowed.